src/main/java/com/nirima/jenkins/plugins/docker/utils/PortUtils.java

```diff
@@ -39,7 +41,7 @@ public static ConnectionCheck connectionCheck(InetSocketAddress address) {
         private long retryDelay = (long) SECONDS.toMillis(2);
 
         private ConnectionCheck(String host, int port) {
-            this.host = host;
+            this.host = "127.0.0.1";//host;
```

Comment:
This will only work if Jenkins is running on the same host as the docker daemon. It won't work in the general case where Jenkins is on a separate host from the docker hosts/swarms that it's talking to.